portfolio/src/main/webapp/index.html

```diff
@@ -22,6 +22,7 @@
   <!-- Main Section -->
   <a id="home"></a>
   <div class="main-info-section">
+    <div id="quote-container"><i></i></div>
```

Comment:
addQuoteToDom would have overwritten this `<i></i>` tag, so your quotes wouldn't show up as italicized.  Better approach would be to use css and attach the style to `quote-container`.